src/app/WebViewImpl.qml

```diff
@@ -61,8 +61,6 @@ WebView {
     /*experimental.certificateVerificationDialog: CertificateVerificationDialog {}
     experimental.proxyAuthenticationDialog: ProxyAuthenticationDialog {}*/
 
-    signal showDownloadDialog(string downloadId, var contentType, var downloader, string filename, string mimeType)
```

Comment:
417, 444, 460 was adapted in the last commit

402 Actions.OpenLinkInNewPrivateWindow, that is not available for webapps (incognito mode)
422 / 711 : (share) maybe we can make a flag --enable-share for the webapp, because many webapps won't have the right permissions for that

Reply:
enabling share would require more changes, if I just remove the "!isWebApp", there is the error `file:///usr/share/morph-browser/WebViewImpl.qml:424: TypeError: Property 'shareLinkRequested' of object WebApp_QMLTYPE_77(0x12cd820, "webappBrowserView") is not a function` in the log